### ISSUE TITLE
#-1923335426------------- Links must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#508]
<strong>Source:</strong>`<a aria-label="" aria-live="polite" class="button-padding--shrunk icon-end-empty" href="https://docdev.arcgis.com/en/iot/analyze/perform-real-time-analysis.htm" name="false" target="_blank" type="false">`
<strong>Element Location: </strong>["#help-button-analytics\\.real-time","a[name=\"false\"]"]
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attributeTags: [Ljava.lang.String;@42150951
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
